### PR TITLE
[Equipe 4] Début de l'implémentation des contrats cadres et du stock

### PR DESCRIPTION
**Pull request by KelianB-IMTA**

### CODE REVIEW

**Comment by massimotisi** [comment]
please remove `CACAO2019/.gitignore`
please cancel the changes you made to `.gitignore`

**Comment by massimotisi** [comment]
Can't accept because it seems to freeze the simulation.

**Comment by massimotisi** [comment]
Almost good.
You correctly removed the gitignore file in the CACAO2019 folder.
But you removed also the gitignore file in the project root. You should not have removed that. Please restore it. You can find it here: https://github.com/cacao2019/CACAO2019/blob/master/.gitignore. Save it, then move it in Eclipse -> git perspective -> CACAO2019 -> Working Tree.
Please interact with @ErineDupont that has the same problem.

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,4 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+/bin/
```
Comment (massimotisi) [inline .gitignore]: Please remove this line

Comment (massimotisi) [reply .gitignore]: only the line /bin/

**Comment by KelianB-IMTA** [comment]
The .gitignore should be fine now

**Comment by massimotisi** [comment]
@KelianB-IMTA can you just add a new empty line at the end of .gitignore?

**Comment by massimotisi** [comment]
I'm made the change myself. I'm merging!

**Pull request merged**